### PR TITLE
kernel/fragments: support loading kernel modules

### PR DESCRIPTION
**Pull request by devimc**

### CODE REVIEW

File: kernel/build-kernel.sh [hunk 1]
```diff
@@ -165,6 +166,11 @@ get_kernel_frag_path() {
 	# here.
 	local all_configs="${common_configs} ${arch_configs}"
 
+	if [ -d "${modules_path}" ]; then
+		modules_config="$(ls ${modules_path}/*.conf)"
```
Comment (jodh-intel) [inline kernel/build-kernel.sh]: Not: `local modules_config`.

**Comment by mythi** [comment]
@eadamsintel FYI

**Comment by jodh-intel** [comment]
@devimc - this one is also conflicted.

**Pull request closed without merging**